src/Azure/README.md

```diff
@@ -0,0 +1,59 @@
+[![Releases](https://img.shields.io/badge/releases-purple)](https://github.com/opentelemetry-php/contrib-aws/releases)
+[![Issues](https://img.shields.io/badge/issues-pink)](https://github.com/open-telemetry/opentelemetry-php/issues)
+[![Source](https://img.shields.io/badge/source-contrib-green)](https://github.com/open-telemetry/opentelemetry-php-contrib/tree/main/src/Azure)
+[![Mirror](https://img.shields.io/badge/mirror-opentelemetry--php--contrib-blue)](https://github.com/opentelemetry-php/contrib-azure)
+[![Latest Version](http://poser.pugx.org/open-telemetry/contrib-azure/v/unstable)](https://packagist.org/packages/open-telemetry/contrib-azure/)
+[![Stable](http://poser.pugx.org/open-telemetry/contrib-azure/v/stable)](https://packagist.org/packages/open-telemetry/contrib-azure/)
+
+This is a read-only subtree split of https://github.com/open-telemetry/opentelemetry-php-contrib.
+
+# OpenTelemetry Azure Resource Detectors
+
+This package provides OpenTelemetry `ResourceDetector`s which will detect 
+resource attributes for these Azure services:
+* App Service
+* Container Apps
+* Virtual Machines
+
+The following OpenTelemetry resource attributes will be detected:
+
+| Resource attribute      | VM       | App Service       | Containers           |
+| cloud.platform          | azure_vm | azure_app_service | azure_container_apps |
+| cloud.provider          | azure    | azure             | azure                |
+| cloud.resource.id       | auto     | auto              |                      |
+| cloud.region            | auto     | auto              |                      |
+| deployment.environment  |          | auto              |                      |
+| host.id                 | auto     | auto              |                      |
+| host.name               | auto     |                   |                      |
+| host.type               | auto     |                   |                      |
+| os.type                 | auto     |                   |                      |
+| os.version              | auto     |                   |                      |
+| azure.vm.scaleset.name  | auto     |                   |                      |
+| azure.vm.sku            | auto     |                   |                      |
+| service.name            |          | auto              | auto                 |
+| service.version         |          |                   | auto                 |
+| service.instance.id     |          | auto              | auto                 |
+| azure.app.service.stamp |          | auto              |                      |
+
+## Requirements
+
+* OpenTelemetry SDK
+
+## Installation via composer
+
+```bash
+$ composer require open-telemetry/contrib-azure
+```
+
+## Usage
+
+The detector will be automatically registered as part of composer autoloading.
```

Comment:
I think this might have been copied over from the `open-telemetry/detector-container` package, so this is not currently the case for this Azure package. Looks like you would just need to copy the `_register.php` logic across to achieve the same!

Reply:
@matt-hensley final bit that makes the register call work is by adding to autoloading mechanism in composer (eg. [here](https://github.com/open-telemetry/opentelemetry-php-contrib/blob/fad23386904c3a5c6e0d9b669e0b0d248f86dc5b/src/ResourceDetectors/Container/composer.json#L18-L21)).